db/index.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/fec', { useNewUrlParser: true});
```

Comment:
is useNewUrlParser necessary for this project?

Reply:
Daniel,
useNewUrlParser seems to be recommended to avoid any deprecation warning.